steel-research-international.csl

```diff
@@ -0,0 +1,220 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" default-locale="en-US">
+  <info>
+    <title>Steel Research International</title>
+    <title-short>Steel Res. Int.</title-short>
+    <id>http://www.zotero.org/styles/steel-research-international</id>
+    <link href="http://www.zotero.org/styles/steel-research-international" rel="self"/>
+    <link href="https://onlinelibrary.wiley.com/page/journal/1869344x/homepage/author-guidelines" rel="documentation"/>
+    <link href="http://www.zotero.org/styles/taylor-and-francis-national-library-of-medicine" rel="template"/>
+    <author>
+      <name>Christian Schubert</name>
+    </author>
+    <contributor>
+      <name>Christian Schubert</name>
+      <email>5238962+c-schubert@users.noreply.github.com</email>
+    </contributor>
+    <category citation-format="author-date"/>
+    <category field="engineering"/>
+    <eissn>1869-344X</eissn>
+    <issnl>1611-3683</issnl>
+    <summary>Citation style for the Steel research international
+    
+    There are some difficulties i.e. with patents, preprint  specifications and the variables that the CSL offers (nor preprint type or company variable availably), which probably cannot be fixed right now (perfectly) and need manual correction.
+    Also there is no way to automatically use or define journal or conference abbrevitation in the csl file right now?</summary>
+    <updated>2021-11-10T16:22:30+01:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <locale xml:lang="en-US">
+    <date form="text" delimiter=" ">
+      <date-part name="month" form="short" strip-periods="true"/>
+      <date-part name="year" font-weight="bold"/>
+    </date>
+    <terms>
+      <term name="chapter" form="short">Ch.</term>
+      <term name="editor" form="short">eds</term>
+    </terms>
+  </locale>
+  <macro name="author">
+    <names variable="author">
+      <name sort-separator=" " initialize-with="." delimiter=", " delimiter-precedes-last="always"/>
+      <substitute>
+        <names variable="editor"/>
+      </substitute>
+    </names>
+  </macro>
+  <macro name="title">
+    <choose>
+      <if type="book">
+        <text variable="title" font-style="italic"/>
+      </if>
+      <else-if type="chapter">
+        <text term="in" suffix=" "/>
+        <text variable="title" font-style="italic"/>
```

Comment:
This can't be right. Should this be container-title?

Reply:
This works as aspected for my zotero test entry:

![image](https://user-images.githubusercontent.com/5238962/142842976-3111c20f-6c63-4f09-b77a-c494157fc5a6.png)
![image](https://user-images.githubusercontent.com/5238962/142843019-7af2d91c-04b8-49e3-8041-b72473825aa1.png)

so I guess `title` is fine?

Reply:
As expected how? The code.produces "in: (chapter) title" which makes no sense for a book section, since "in" indicates the book/container an item was published in.

Reply:
Ah OK, sorry my bad `container-title` is totally right. I oversaw that there is an extra booktitle option in zotero ...